Encounters.md

```diff
@@ -29,6 +35,17 @@ In addition to requiring sufficient information to create an associated [Individ
   > Note: while not required to create an encounter, `nasal_swab_collection` must be complete and verified to link an associated [Sample].
 
 ### SFS
+- [UW Reopening]
+  - `eligibility_screening` is complete and verified
+  - `consent_form` is complete and verified
+  - `enrollment_questionnaire` is complete and verified
+  - `parental_consent_form` is complete if participant < 18 years old
```

Comment:
The ETL also checks that the `signature_parent` field is not blank

Reply:
Right, but isn't this a bit redundant since `signature_parent` is required in the parental consent form?

Reply:
i.e. we can't have a complete form and have the signature be blank, right?

Reply:
Ah, right it's a required field. Ignore my comment 😄

Reply:
REDCap does not enforce that a required field be populated. When you mark a field as required the UI will prompt the user if they haven't set a value, but it does allow the user to set the survey as complete and to save. I just did this with the parent consent form in the test Husky project.

From https://wiki.uiowa.edu/display/REDCapDocs/Handling+Blank+Fields
"
Once this option is set, the user will be prompted with a warning dialog if the required field is left blank. This will make it very difficult for a survey participant to complete the survey without first entering a value in this field.
Please Note
Please remember that since REDCap is a web application, there is no way to guarantee that a field will NOT be skipped.  You simply can not force the end user to enter a data value if they refuse to do so.  However, using required fields and the other techniques mentioned below will "strongly encourage" compliance. 
"

I wanted to make sure that we had parental consent. This is why I check the signature field as well as the complete status.

Reply:
Oh, thanks for digging into that, @jccraft-uw ! I will update the wiki to reflect that the signature field needs to be complete, too.